js/utility/read-file.js

```diff
@@ -18,6 +18,9 @@ define([
     var readFileAsJson = (path) => {
         return readFile(path)
         .then((data) => {
+            if (data !== null && typeof data === 'object' && typeof data !== 'string') {
+                return data;
+            }
             return JSON.parse(data);
```

Comment:
Also try catch, in case the json is not valid

Reply:
if the json is not valid it cannot run, no need to fail safely.